src/create_venv.sh

```diff
@@ -1,136 +1,258 @@
 #!/usr/bin/env bash
 
+# Copyright (c) YugaByte, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied.  See the License for the specific language governing permissions and limitations
+# under the License.
+#
 
-# Use SHA of requirements.txt as a comment in requirements_frozen.txt
-# 
-
-function text_file_sha() {
-  local file="${1}"
-  local tmp
-  tmp="$(sort -u <<<"$(grep -v '^#' "${file}")")"
-  awk '{print $1}'<<<"$(sha256sum <<<"${tmp}")"
-}
+[[ "${_YB_CREATE_VENV_INCLUDED:=""}" == "yes" ]] && return 0
+_YB_CREATE_VENV_INCLUDED=yes
+YB_VERBOSE=${YB_VERBOSE:-false}
```

Comment:
YB_VERBOSE is a generic utility and does not belong in a venv-specific library file.